tests/functional/gluster_stability/test_gluster_log_rotation.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+import re
+
+import pytest
+
+from openshiftstoragelibs import baseclass
+from openshiftstoragelibs import command
+from openshiftstoragelibs import exceptions
+from openshiftstoragelibs import openshift_ops
+from openshiftstoragelibs import waiter
+
+TCMU_CONF = "/etc/tcmu/tcmu.conf"
+TCMU_RUNNER_LOG = "/var/log/glusterfs/gluster-block/tcmu-runner.log"
+TCMU_ROTATE_LOG = "/etc/logrotate.d/gluster-block"
+# RE to check time and date from logs
```

Comment:
This is logically different block. Add extra line before the comment.

Reply:
Done